projects/question/src/main/java/org/batfish/question/ReachabilityQuestionPlugin.java

```diff
@@ -523,6 +552,16 @@ public void setFinalNodeRegex(String regex) {
       _finalNodeRegex = regex;
     }
 
+    @JsonProperty(PROP_TRANSIT_NODES)
+    public void setTransitNodes(SortedSet<String> transitNodes) {
+      _transitNodes = transitNodes;
+    }
+
+    @JsonProperty(PROP_NOT_TRANSIT_NODES)
+    public void setNotTransitNodes(SortedSet<String> notTransitNodes) {
+      _notTransitNodes = notTransitNodes;
+    }
+
```

Comment:
Can you please add tests of these new features to the example tests? 

Probably, just go in [here](https://github.com/batfish/batfish/blob/master/tests/basic/commands#L35) similar to what you wrote in the PR desc, with new ref files.